### PR TITLE
added xScale option for multiChart graph

### LINKED ISSUE #814: Please support xScale in multiChart
I'd like a graph with two Y axes with different scales, and xScale(d3.time.scale.utc()) so I think I need to use multiChart, but this does not I think support xScale yet?

Steve

### PR DESCRIPTION
**Pull request by Robsonsjre**
Added new option `xScale` to multiChart graph solvind the issue 

```
chart = nv.models.multiChart()
      .margin({ right: 23, top: 12, left: 60, bottom: 75 })
      .showLegend(false)
      .xScale(d3.time.scale.utc())
```

This may close #814

### CODE REVIEW

**Comment by mistercrunch** [comment]
@Robsonsjre I was about to write this same PR but saw you already did. Mind rebasing so that maintainers can see a successful build and hopefully merge this?

File: src/models/multiChart.js [hunk 1]
```diff
@@ -538,6 +538,7 @@ nv.models.multiChart = function() {
         width:      {get: function(){return width;}, set: function(_){width=_;}},
         height:     {get: function(){return height;}, set: function(_){height=_;}},
         showLegend: {get: function(){return showLegend;}, set: function(_){showLegend=_;}},
+        xScale: {get: function(){return x;}, set: function(_){ x = _; xAxis.scale(x); }},
```
Comment (liquidpele) [inline src/models/multiChart.js]: Isn't this already possible by doing chart.xAxis.scale(...), since xAxis is made available on the main chart object?

Comment (mistercrunch) [inline src/models/multiChart.js]: when I do `chart.xAxis.scale(d3.time.scale.utc());` I get 
screen shot 2017-08-23 at 11 48 02 pm

Comment (liquidpele) [inline src/models/multiChart.js]: Hmm, weird... well, adding this option doesn't really hurt anything so I'll merge it.

Comment (liquidpele) [inline src/models/multiChart.js]: Oh wait, it looks like unittests failed...  sometimes (constantly) traviceCI bombs out though, if you run them yourself does it also fail?

Comment (liquidpele) [inline src/models/multiChart.js]: I can't imagine your change broke anything, so I'll just merge.

Comment (mistercrunch) [inline src/models/multiChart.js]: I hate when tests just randomly start failing. Thanks for taking the time to review & merge this!

**Comment by gmatheus** [comment]
@liquidpele Is this already in the latest release? I've installed the 1.8.6 version (from [npm](https://www.npmjs.com/package/nvd3)) and it seems that `xScale` still doesn't work for `multiChart`.

**Comment by liquidpele** [comment]
Try the code in the latest master branch.  You may need to create the build files yourself via "grunt production"

**Pull request merged**